pkg/compat/metrics.go

```diff
@@ -25,15 +26,38 @@ const (
 )
 
 //
-// Global reporter.
-var Metrics *Reporter
+// Numeric metrics consts
+const (
+	one         = 1
+	nanoToMilli = 1e6
+)
+
+//
+// Global reporters.
+var RequestCountMetrics *Reporter
+var RequestRTTMetrics *Reporter
 
 func init() {
-	Metrics = &Reporter{
+	RequestCountMetrics = &Reporter{
+		counter: promauto.NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "mtc_client_count",
```

Comment:
I think I need to make this name more descriptive. The consumer of this data will not have our code for context.